gradient_descent.py

```diff
@@ -85,14 +85,22 @@ def squeeze_into_ranges(point, ranges, padding=0.0, verbose=False):
     for key in ranges:
         if len(ranges[key]) < 2:
             continue
-        if point[key] < ranges[key][0]:
+
```

Comment:
in ranges, you no longer have to give the smallest value first. From now on, negative pressure amplitudes, like this: `ranges = dict(..., p_A=[-1e5*x for x in [0.5, 4.0]])` won't throw errors